docs/mine/lotus/split-markets-miners.md

```diff
@@ -68,51 +104,66 @@ export LOTUS_BACKUP_BASE_PATH=/tmp
 lotus-miner run
 ```
 
-## Splitting the `lotus-miner` monolith
-
-### 1. Creating a backup
+#### 1. Create a backup
 
 ```shell
 export LOTUS_BACKUP_BASE_PATH=/tmp
 lotus-miner backup /tmp/backup.cbor
 ```
 
```

Comment:
Do we need to restart the nodes without `LOTUS_BACKUP_BASE_PATH`?

Reply:
I don't believe it matters